### PR TITLE
fix(dbt-fal): make dbt-fal work with redshift

### PR DESCRIPTION
**Pull request by mederka**
- Fixes installation of dbt-fal with `pip install dbt-fal[redshift]`
- Adds redshift to integration tests

### CODE REVIEW

File: .github/workflows/test_integration_adapter.yml [hunk 1]
```diff
@@ -64,6 +65,11 @@ jobs:
             pip install 'dbt-${{ matrix.profile }}==${{ matrix.adapter_version }}'
           fi
 
+          if [[ '${{ matrix.profile }}' == "redshift" ]]
+          then
+            pip install dbt-postgres
+          fi
+
```
Comment (chamini2) [inline .github/workflows/test_integration_adapter.yml:72]: I think we don't have to install it, it's a dependency: https://github.com/dbt-labs/dbt-redshift/blob/main/setup.py#L70

File: adapter/pyproject.toml [hunk 2]
```diff
@@ -46,6 +48,7 @@ bigquery = ["google-cloud-bigquery"]
 redshift = ["sqlalchemy-redshift"]
 duckdb = []
 athena = []
+teleport = ["s3fs"]
```
Comment (chamini2) [inline adapter/pyproject.toml:51]: Can we make a ticket to

```py
yield 'dbt-fal[<adapter>,teleport]', ...
```

when using teleport?

Comment (mederka) [reply adapter/pyproject.toml:51]: We have a "test teleport" ticket, and I'm sure we will resolve this issue there.

File: .github/workflows/test_integration_adapter.yml (hunk 1 above)
Comment (mederka) [reply .github/workflows/test_integration_adapter.yml]: Good catch!

**Pull request merged**